packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -631,11 +690,44 @@ exports.extendNodeType = ({ type, store, cache, getNodesByType }) => {
   const sizesNode = fluidNodeType({ name: `ContentfulSizes`, getTracedSVG })
   sizesNode.deprecationReason = `Sizes was deprecated in Gatsby v2. It's been renamed to "fluid" https://example.com/write-docs-and-fix-this-example-link`
 
+  // gatsby-pugin-image
+  const getGatsbyImageData = () => {
+    // @todo check if gatsby-plugin-image is enabled
```

Comment:
How can I check if gatsby-plugin-image is enabled?

Reply:
You could look for a SItePlugin node with a matching name. I'd memoize it if you do that